src/serlo-editor-repo/plugin-text/components/text-editor.tsx

```diff
@@ -98,17 +96,23 @@ export function TextEditor(props: TextEditorProps) {
 
   // Workaround for setting selection when adding a new editor:
   useEffect(() => {
-    // If the editor is not focused, do nothing
-    if (focused === false) return
+    // Get the current text value of the editor
+    const text = Node.string(editor)
+
+    // If the editor is not focused, remove the suggestions search
+    // and exit the useEffect hook
+    if (focused === false) {
+      if (text.startsWith('/')) {
```

Comment:
> Is it possible to check if the popover is still open at this point?

The popover will be closed at this point, because it's only open when `focused === true`.

This logic will change, however. I'll open an issue for that now.